elements/lisk-chain/src/state_store/account_store.ts

```diff
@@ -82,16 +60,15 @@ export class AccountStore {
 		}
 
 		// Account was not cached previously so we try to fetch it from db
-		const [elementFromDB] = await this._account.get(
-			{ [this._primaryKey]: primaryValue },
-			{ limit: null },
+		const elementFromDB = await this._dataAccess.getAccountByAddress(
```

Comment:
shouldn't be better to check first its existance in `this._data` and if not there then hit the storage?

Reply:
it's not changed, but it's checking in `this._data` above